docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2811,6 +2818,12 @@ const participantsList = document.querySelector('#participant-list');
 const participantTable = document.querySelector('#participant-table');
 const participantButtons = document.querySelector('#participant-btn');
 
+participantTable.addEventListener('click', (event) => {
+  const selectedParticipant = meeting.members.membersCollection.get(event.target.id);
+  const muteButton =  document.querySelector('#participant-btn .btn-group button:first-child');
+  muteButton.innerText = selectedParticipant.isAudioMuted ? 'Unmute': 'Mute';
```

Comment:
I thought if the mute state is already muted, we will just disable the button because Host can only Mute a participant but cannot unmute

Reply:
We cannot unmute but there is a request send to unmute which is visible on the other participant's event list hence I didn't want to break that functionality. We can show one message to indicate on the host that unmute request has been sent to make it more clear

Reply:
In that case, can we make this button "Request to unmute" instead of just "Unmute"?

Reply:
Changed test to -> 'Request to unmute'

Reply:
There are scenarios where Unmute is possible to be done so keeping it Request To unmute wouldn't be always correct. Updating it to Toggle Audio now

Reply:
but "toggle audio" doesn't feel correct. In most cases when someone muted themselves and you press "toggle audio" their audio won't be toggled, only the request to unmute will be sent, right?

Reply:
yes that's correct